frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -177,6 +178,134 @@ export function ReplayAuthorizedDomains(): JSX.Element {
+    }
+    const currentConfig: SessionRecordingSummaryConfig = currentTeam.session_recording_summary_config || defaultConfig
+
+    return (
+        <div className="flex flex-col gap-2">
+            <div>
+                <LemonButton
+                    type="secondary"
+                    onClick={() => {
+                        updateCurrentTeam({
+                            session_recording_summary_config: defaultConfig,
+                        })
+                    }}
+                >
+                    Reset to default
+                </LemonButton>
+            </div>
+            <div>
+                <p>
+                    We use Open AI to summarise sessions. No data is sent to OpenAI without an explicit instruction to
+                    do so. Only by clicking the <IconAutoAwesome /> "Summary" button will selected event data be shared
+                    with a third party. We only send the data selected below.{' '}
+                    <strong>Data submitted is not used to train Open AI's models</strong>
+                </p>
+                <LemonSwitch
+                    checked={currentConfig.opt_in}
+                    onChange={(checked) => {
+                        updateCurrentTeam({
+                            session_recording_summary_config: {
+                                ...currentConfig,
+                                opt_in: checked,
+                            },
+                        })
+                    }}
+                    label="Opt in to enable AI suggested summaries"
+                />
+            </div>
+            <div>
+                <h3 className="flex items-center gap-2">
+                    <IconSelectEvents className="text-lg" />
+                    Preferred events
+                </h3>
+                <p>
+                    These events are treated as more interesting when generating a summary. We recommend you include
+                    events that represent value for your user
+                </p>
+                <EventSelect
+                    onChange={(includedEvents) => {
+                        updateCurrentTeam({
+                            session_recording_summary_config: {
+                                ...currentConfig,
+                                preferred_events: includedEvents,
+                            },
+                        })
+                    }}
+                    selectedEvents={currentConfig.preferred_events || []}
+                    addElement={
+                        <LemonButton size="small" type="secondary" icon={<IconPlus />} sideIcon={null}>
+                            Add event
+                        </LemonButton>
+                    }
+                />
+            </div>
+            <div>
+                <h3 className="flex items-center gap-2">
+                    <IconSelectEvents className="text-lg" />
+                    Excluded events
```

Comment:
My worry with the "Excluded events" approach is that the data will be sent until it is explicitly excluded. It might not be the worst impact given the event name is all that is sent by default (as properties have to be allow listed) but it's not the most intuitive for someone adding a new event to think "I wonder if this should be excluded from the AI summary feature"